dockerfiles.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+TEMPLATE="Dockerfile.in"
+RELEASES="v3.6 v3.7 v3.8 v3.9 edge"
+ARCHS="x86 x86_64 armhf armv7 aarch64"
+
+die() {
+	echo "$1" >&2
+	exit 1
+}
+
+arch_to_image() {
+	case $1 in
+		armv7) echo arm32v7 ;;
+		aarch64) echo arm64v8 ;;
+		x86_64) echo amd64 ;;
+		armhf) echo arm32v6 ;;
+		x86) echo i386 ;;
+		*) die "Unknown arch detected: \"$1\""
+	esac
+}
+
+[ -d "Dockerfiles" ] && rm -rf Dockerfiles/*
```

Comment:
```suggestion
[ -e "Dockerfiles" ] && rm -rf Dockerfiles/*
```

You want to remove a file called "Dockerfiles" too, or the mkdir will fail.